website/content/docs/getting-started.md

```diff
@@ -178,6 +178,8 @@ You can open Heron UI in your browser at [http://localhost:8889](http://localhos
 and see something like this upon successful submission of the topology:
 ![alt tag](/img/heron-ui.png)
 
+Note: ``heron-tracker`` (in Step 4) should be running at the same time so that Heron UI can work properly. 
```

Comment:
The first sentence section of this section is `Heron UI is a user interface that uses Heron Tracker`. The text is very clear IMO that the tracker needs to be running.